README.md

```diff
@@ -74,12 +74,12 @@ detailed comparison of `TypedDataset` with Spark's `Dataset` API.
 * [Proof of Concept: TypedDataFrame](http://typelevel.org/frameless/TypedDataFrame.html)
 
 ## Quick Start
-Frameless is compiled against Scala 2.11.x (and Scala 2.12.x since Frameless 0.8.0)
+Frameless is compiled against Scala 2.12.x
```

Comment:
Let's mention that "Since the 0.9.x release, Frameless is compiled only against Scala 2.12.x"

Reply:
changed.